vertx-grpc-health/pom.xml

```diff
@@ -85,20 +85,29 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.xolstice.maven.plugins</groupId>
+        <groupId>io.github.ascopes</groupId>
         <artifactId>protobuf-maven-plugin</artifactId>
         <executions>
           <execution>
             <id>test-compile</id>
             <goals>
-              <goal>test-compile</goal>
-              <goal>test-compile-custom</goal>
+              <goal>generate-test</goal>
             </goals>
+            <configuration>
+              <binaryMavenPlugins>
+                <binaryMavenPlugin>
```

Comment:
I think this section should be defined in a pluginManagement section of the parent pom file and avoid to be part of every sub project